src/runner/sqs.ts

```diff
@@ -203,7 +218,7 @@ class SqsRunner extends BaseRunner implements IRunner {
         MessageRetentionPeriod: messageRetentionPeriod,
       },
     };
-    return this.sqs.createQueueAsync(params);
+    await this.sqs.createQueue(params).promise();
```

Comment:
Any particular reason to move away from just returning?

Reply:
@rrsmith the type on the base runner is void, so this was triggering the type check

Reply:
Ahhh, make sense